extensions/amp-script/amp-script.md

```diff
@@ -23,37 +23,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# amp-script
+The `amp-script` component allows you to run custom JavaScript. To maintain AMP's performance guarantees, your code runs in a Web Worker, and certain restrictions apply.
```

Comment:
Can we move this right under `## Usage`?

Reply:
I was wondering about that, since it would meet [your new standards](https://github.com/ampproject/amphtml/blob/master/build-system/tasks/extension-generator/extension-doc.template.md). I only held back because other docs had this brief summary before the first header.

Made the change, thanks!